src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,3 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/marketplace_product?currentSchema=public
+spring.datasource.username=market_user
+spring.datasource.password=password
```

Comment:
best practice is to retrieve them from env variables and use a default if env variables not set. Can you also change file name to application-local.properties. It's more clear that way.

```
spring.datasource.username=${DATASOURCE_USERNAME:market_user}
spring.datasource.password=${DATASOURCE_PASSWORD:password}
```